backend/app/views/spree/admin/promotions/_form.html.erb

```diff
@@ -66,12 +66,14 @@
 
     <div id="starts_at_field" class="field">
       <%= f.label :starts_at %>
-      <%= f.text_field :starts_at, :value => datepicker_field_value(@promotion.starts_at), :class => 'datepicker datepicker-from fullwidth' %>
+      <%= f.field_hint :starts_at %>
+      <%= f.text_field :starts_at, :value => datepicker_field_value(@promotion.starts_at), :placeholder => "Immediately", :class => 'datepicker datepicker-from fullwidth' %>
```

Comment:
Please pass the placeholder value through `I18n.t` and preferable use a key with a leading period, so the key gets scoped to this particular view.